gui/install.sh

```diff
@@ -35,23 +37,43 @@ modify_locations() {	# Some files have placeholders for certain locations.  Modi
 }
 
 NEED_TO_UPDATE_HOST_NAME="true"
-
 CURRENT_HOSTNAME=`cat /etc/hostname | tr -d " \t\n\r"`
 
 # Check if the user is updating an existing installation.
 if [ "${1}" = "--update" -o "${1}" = "-update" ] ; then
-	UPDATE="true"
 	shift
+
+	echo -en '\n'
+	echo -e "*********************************************"
+	echo    "*** Performing update of the Allsky WebUI ***"
+	echo -e "*********************************************"
+	echo -en '\n'
+
 	if [ ! -d "${PORTAL_DIR}" ]; then
 		echo -e "${RED}Update specified but no existing WebUI found in '${PORTAL_DIR}'${NC}" 1>&2
 		exit 2
 	fi
 
 	modify_locations
+
+	# Add entry to sudoers file if not already there.
+	# This is only needed for people who updated allsky-portal but didn't update allsky.
+	# "vcgencmd" added to sudoers on November 11, 2021
```

Comment:
remove this

Reply:
Chris, I'm not sure what change(s) you're suggesting.

Basically, the code is trying to catch the case were someone grabs the updated "system.php" and "gui/install.sh" files but not the "gui/sudoers" file.  When they click on the "System" link in the WebUI they'll see a message to execute "gui/install.sh --update".  When they do that the code will check if the "sudoers" file is updated; if not the user will get a message to update it.  If it IS updated, it will be written to /etc/sudoers.d.

I could skip the check to see if the /etc/sudoers.d/allsky file is already updated, but I think that's bad coding practice.  There's no harm in checking, and IMHO it makes the code more obvious what it's doing and why.

Reply:
Is now `grep --silent "/usr/bin/vcgencmd" /etc/sudoers.d/allsky`